docs/datatypes/tpromise.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: datatypes_tpromise
-title: "TPromise"
+title:  "TPromise"
```

Comment:
Are the extra spaces on purpose?

Reply:
@mschuwalow thanks for the review. 

Not sure, I've just tried to be consistent with other files in `datatypes` directory.